docs/products/ESPHome-Starter-Kit/modules/motion-module.md

```
@@ -117,4 +117,13 @@ With the device back online, the Motion entity is live on the web server. <a hre
 
 !!! success "Your Motion module is now ready to use in automations!"
 
-    You're now ready to make automations around occupancy in your home!
\ No newline at end of file
+    You're now ready to make automations around occupancy in your home!
+
+## Try it in an automation
+
+Your motion sensor is live. Put it to work:
+
+<a href="/products/ESPHome-Starter-Kit/automations/motion-activated-light/" class="md-button md-button--primary"> <img src="/assets/esphome-logo.svg" /> Motion-Activated Light </a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)